lib/api/contacts.js

```diff
@@ -3,3 +3,71 @@ export const queryEmails = (data) => ({
     method: 'get',
     data
 });
+
+export const getContacts = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
```

Comment:
Use `query` instead of `get` when it returns an `Array`.
```suggestion
export const queryContacts = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
```

Reply:
Done!